### ISSUE TITLE
3.0 - Reports Screens duplicate ID elements

### ISSUE THREAD

**Issue by robincornett** [easydigitaldownloads/easy-digital-downloads#7976]
### Expected behavior
ID attributes for HTML elements should be unique.

### Actual behavior
On the reports screens, each chart/graph is output with the same ID (`edd-reports-charts-wrap`) and class (`edd-report-wrap`). This presents accessibility issues, and potentially styling issues, since pie charts and line graphs have the same ID and class.

### Steps to reproduce the behavior
1) Visit Downloads > Reports
2) Compare the IDs on the Sales and Earnings chart and the Refunds charts. They are identical.

### Information (if a specific version is affected):
PHP Version: 7.3.5
EDD Version (or branch): 3.0
WordPress Version: 5.4.1

Any other relevant information:
Report tables and tiles do not appear to be affected by this. All tiles are in one container with an ID of `edd-reports-tiles-wrap`, and each tile has a unique ID. Where multiple tables are displayed, they are in one container (`#edd-reports-tables-wrap`) as well, and have unique IDs.

Based on the tiles and tables output, I suspect that multiple charts/graphs should be output in one `#edd-reports-charts-wrap` container, with another container for each individual chart, using unique IDs (probably the `object_id`), in order to fix this issue and be consistent with other reports output.
